### DISCUSSION TITLE
Multiple targets mode for LoadBot

### DISCUSSION THREAD

**Discussion by kirillt** [0xPolygon/polygon-edge#502] [Ideas]
#feature #question
Would it be useful to implement remote mode for the stress testing module?

This would look like an extra flag `--targets` receiving a list of `IP:PORT` pairs and sending transactions to all specified nodes in round-robin or random way. Such feature should provide less "stressful" but more realistic results since load would be evenly distributed between many nodes.

It might be also a good idea to separate the tool into separate module in this case, since machine used for stress test invocation doesn't have to run blockchain node itself.

**Comment by zivkovicmilos** [comment]
Hey @kirillt,

Thank you for raising this suggestion. What kinds of features would you expect from a loadbot that's a separate module?

I think it's a cool idea, and something we've been definitely looking into. As such, we would love to hear your inputs on how we can better shape the loadbot Polygon Edge offers.

**Comment by kirillt** [comment]
Hi @zivkovicmilos I really like LoadBot's interface, I don't have any ideas what could be improved about it except remote invocation. Just list of target nodes as a parameter and also a switch between random and round-robin modes would be nice.
